layers/+window-management/exwm/files/exwm-start

```diff
@@ -5,19 +5,10 @@ xhost +
 ## you might need to append the TTY you are working on
 xinit
 
-wmname LG3D
-
-# Remap caps lock to left control. This is not strictly speaking
-# exwm related, but it's handy
-setxkbmap -option 'ctrl:no caps'
+wmname EXWM
 
 # Set fallback cursor
 xsetroot -cursor_name left_ptr
 
-# If Emacs is started in server mode, `emacsclient` is a convenient way to edit
-# files in place (used by e.g. `git commit`)
-export VISUAL=emacsclient
-export EDITOR="$VISUAL"
```

Comment:
I'm setting the defaults here as per: https://github.com/ch11ng/exwm/wiki/Configuration-Example

Can we keep it as recommended by the author of EXWM unless there is a good reason not to?

Reply:
Presumably all people using Spacemacs will want to use it always.
Maybe the desired behavior will be something like `emacsclient -c` instead.

I think for the same reasons as the keyboard remapping this should be left out of the default config.

Maybe I can add a configuration option?